### DISCUSSION TITLE
Evaluation metrics for top-down trained model

### DISCUSSION THREAD

**Discussion by ozgurcemozgen06** [talmolab/sleap#1298] [Help!] [answered]
Hi all, 

I just want to ask something regarding the top-down trained model on SLEAP. In order to inspect the evaluation metrics for top-down model built on SLEAP, which resultant metrics should we use, since there are two directories created under /models directory? 

Kind regards,
Özgür

**Comment by talmo** [comment, marked as answer]
Hi @ozgurcemozgen06,

Great question. They each give you a different view on the metrics, but they're essentially the isolated performance of each model when using the ground truth for the other.

The centered instance model's performance is almost certainly what you want since it captures landmark localization performance. If you'd like to get the overall performance with both models you can [check out this notebook](https://sleap.ai/notebooks/Model_evaluation.html) for how to run your own evaluation.

Cheers,

Talmo
